scripts/lc-builds/toss3_hipcc.sh

```diff
@@ -22,7 +22,7 @@ COMP_VER=$1
 COMP_ARCH=$2
 
 BUILD_SUFFIX=lc_toss3-hipcc-${COMP_VER}-${COMP_ARCH}
-RAJA_HOSTCONFIG=../tpl/RAJA/host-configs/lc-builds/toss3/hip.cmake
+RAJA_HOSTCONFIG=../tpl/RAJA/host-configs/lc-builds/toss3/hip_link_${COMP_VER}.cmake
```

Comment:
Was it that we were using the incorrect cmake script that lead to issues when building on AMD platforms?

Reply:
@artv3 after the recent system upgrade on corona, the code would no longer run when built with the old hip script due to AMD rearranging their runtime library directories. John G. sent me what he claims is the only sane way to link the code. I added that to our host config files.